### DISCUSSION TITLE
Targeted Version?

### DISCUSSION THREAD

**Discussion by I2pRandom** [TeamGalacticraft/Galacticraft#345] [Q&A]
Is this repository actually Targeting a minecraft version, or is it just Latest Version chasing like the mod Arcana did until all the devs burned themselves out, and Arcana was canceled?

**Comment by TheMCSpacePro** [comment]
I think it's targeting the newest version of Minecraft.

**Reply by I2pRandom** [reply]
Really hope this mod doesn't burn it's self out before it even gets off the ground if it is.

**Comment by EzerArch** [comment]
_(While I'm aware I've been away for long...)_

> or is it just Latest Version chasing like the mod Arcana did until all the devs burned themselves out...?

Interestingly, this is what I've thought a few times. Targeting at latest MC version _may mean_ that most of that bulk effort will be used in porting efforts, and I do know this is pain in the neck.

BTW, I'm happy to see activity on the development. Cheers to the devs, specially the new ones! :)
